db/sequelize.js

```diff
@@ -37,6 +37,7 @@ if (process.env.POSTGRESQL_SERVICE_PORT) {
     options.port = process.env.DATABASE_PORT;
 }
 
+
```

Comment:
delete

Reply:
This is needed so we can test and also use the db in prod